scripts/windows/msi-install.ps1

```diff
@@ -0,0 +1,39 @@
+param (
+    [int]$Port = 3306,
+    [string]$InstallDir = "C:\Program Files\MariaDB",
+    [string]$ServiceName = "MariaDB"
+)
+
+# Look for the first MariaDB MSI file in the current directory
```

Comment:
Similar suggestion, $BuildDir sounds like a good Param to have. Then the comment can reference BuildDir, not current directory. Makes the script more flexible to where it's run from.